invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml

```diff
@@ -74,7 +74,7 @@
     </datafield>
     <datafield tag="538" ind1=" " ind2=" ">
       <subfield code="a">Use this with a Micro-CernVM available at</subfield>
-      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+      <subfield code="w">http://cernvm.cern.ch/portal/downloads</subfield>
```

Comment:
This change of subfields seems wrong; 538 $w should contain record ID values, see the field rule book:

- https://github.com/cernopendata/opendata.cern.ch/blob/master/invenio_opendata/base/recordext/fields/opendata.cfg#L154-L160